### PR TITLE
query (Top)Schaufensterkonditionen add validation rules auszahlungsbetrag and laufzeit

### PR DESCRIPTION
**Pull request by cryptoki**
query (Top)Schaufensterkonditionen add validation rules auszahlungsbetrag and laufzeit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,17 +77,22 @@ The URL for the Schaufensterkonditionen is:
 
 ### Query Top-Schaufensterkondition
 
+#### Hints
+
+* The calculation of Top-Schaufensterkondition is only possible if there is a valid **Auszahlungsbetrag** and **LaufzeitInMonaten**.
```
Comment (sgrell) [inline README.md:82]:
```suggestion
* The calculation of Top-Schaufensterkondition requires a valid `auszahlungsbetrag` and `laufzeitInMonaten`.
```

File: README.md [hunk 2]
```diff
@@ -145,17 +150,22 @@ The Query returns a [Schaufensterkondition](#schaufensterkondition).
 
 ### Query Schaufensterkonditionen
 
+#### Hints
+
+* The calculation of Top-Schaufensterkondition is only possible if there is a valid **Auszahlungsbetrag** and **LaufzeitInMonaten**.
```
Comment (sgrell) [inline README.md:155]:
```suggestion
* The calculation of Top-Schaufensterkondition requires a valid `auszahlungsbetrag` and `laufzeitInMonaten`.
```

File: README.md [hunk 3]
```diff
@@ -145,17 +150,22 @@ The Query returns a [Schaufensterkondition](#schaufensterkondition).
...
-| partnerId          | [Partner-ID](#partner-id)                 | The Partner-ID of the API Client |
-| auszahlungsbetrag  | Euro!                                     | - (mandatory field)              |
-| laufzeitInMonaten  | Int                                       | -                                |
-| finanzierungszweck | [Finanzierungszweck](#finanzierungszweck) | `FREIE_VERWENDUNG`               |
-| datenkontext       | [Datenkontext](#datenkontext)             | `TESTUMGEBUNG`                   |
+| Parameter Name     | Type                                      | Default Value                       |
+|--------------------|-------------------------------------------|-------------------------------------|
+| partnerId          | [Partner-ID](#partner-id)                 | The Partner-ID of the API Client    |
+| auszahlungsbetrag  | Euro!                                     | - (mandatory field, greater than 0) |
+| laufzeitInMonaten  | Int                                       | Duration in month (greater than 0)  |
```
[the remark this answers is not in the document]
Comment (cryptoki) [reply README.md]: @sgrell I introduced the column `condition`.

**Review by XiangRongLin — DISMISSED** [review]
~~gibts dazu kein jira ticket?~~

File: README.md [hunk 4]
```diff
@@ -77,17 +77,22 @@ The URL for the Schaufensterkonditionen is:
...
 The GraphQL-Query is called `topSchaufensterkondition` and contains the following parameter:
 
-| Parameter Name     | Type                                      | Default Value                                     |
-|--------------------|-------------------------------------------|---------------------------------------------------|
-| partnerId          | [Partner-ID](#partner-id)                 | The Partner-ID of the API Client                  |
-| auszahlungsbetrag  | Euro!                                     | - (mandatory field)                               | 
-| laufzeitInMonaten  | Int                                       | -                                                 | 
-| finanzierungszweck | [Finanzierungszweck](#finanzierungszweck) | Calculation over all values of Finanzierungszweck |
-| datenkontext       | [Datenkontext](#datenkontext)             | `TESTUMGEBUNG`                                    |
+| Parameter Name     | Type                                       | Default Value                                     | Condition      |
```
Comment (sgrell) [inline README.md:89]: Condition is okay, maybe Explanation or Description is more versatile.

File: README.md [hunk 5]
```diff
@@ -77,17 +77,22 @@ The URL for the Schaufensterkonditionen is:
...
+|--------------------|--------------------------------------------|---------------------------------------------------|----------------|
+| partnerId          | [Partner-ID](#partner-id)                  | The Partner-ID of the API Client                  |                |
+| auszahlungsbetrag  | Euro!                                      | - (mandatory field)                               | greater than 0 | 
+| laufzeitInMonaten  | Int                                        |                                                   | greater than 0 |
```
Comment (sgrell) [inline README.md:93]: Even if the datatype is doesn't make it mandatory, we should mark it as mandatory. And I would make the additional text more verbose, just like the rest.
```suggestion
| laufzeitInMonaten  | Int                                        | - (mandatory field)                               | Value must be greater than 0 |
```

File: README.md [hunk 6]
```diff
@@ -145,17 +150,22 @@ The Query returns a [Schaufensterkondition](#schaufensterkondition).
...
-| laufzeitInMonaten  | Int                                       | -                                |
-| finanzierungszweck | [Finanzierungszweck](#finanzierungszweck) | `FREIE_VERWENDUNG`               |
-| datenkontext       | [Datenkontext](#datenkontext)             | `TESTUMGEBUNG`                   |
+| Parameter Name     | Type                                      | Default Value                       | Condition      |
+|--------------------|-------------------------------------------|-------------------------------------|----------------+
+| partnerId          | [Partner-ID](#partner-id)                 | The Partner-ID of the API Client    |                |
+| auszahlungsbetrag  | Euro!                                     | - (mandatory field)                 | greater than 0 |
+| laufzeitInMonaten  | Int                                       |                                     | greater than 0 |
+| finanzierungszweck | [Finanzierungszweck](#finanzierungszweck) | `FREIE_VERWENDUNG`                  |                |
+| datenkontext       | [Datenkontext](#datenkontext)             | `TESTUMGEBUNG`                      |                |
```
Comment (sgrell) [inline README.md:168]: see above

**Review by XiangRongLin — APPROVED** [review]

**Pull request merged**